packages/ui-extensions/src/surfaces/admin/globals.ts

```diff
@@ -14,3 +18,16 @@ declare global {
   // @ts-ignore
   const shopify: ShopifyGlobal;
 }
+
+export interface AdminReferenceEntityTemplateSchema
```

Comment:
We should move this to a separate file. This is the global object that gets exposed to developers using admin-extensions. We shouldn't add doc specific types in here. Maybe a `docs-types.ts` or something.